proto/sdk/alpha/alpha.proto

```diff
@@ -124,7 +135,7 @@ message Count {
 
 // Store a boolean result
 message Bool {
-    bool bool = 1;
+    bool bool = 1 [(grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {format: "boolean"}];
```

Comment:
If `[(grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {format: "boolean"}]` is not defined, the output of swagger.json is missing `format: "boolean"`.